src/timeCalculator.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+
+function TimeCalculator(props) {
```

Comment:
Nice modularization here. Keeps ideas separate so you can focus on one thing at a time when reviewing or debugging.